### PR TITLE
Fix message source modal

### LINKED ISSUE #3604: View source modal is broken
The view source instead of opening as a modal, overlaps the body message

sourcebug

### PR DESCRIPTION
**Pull request by GretaD**
fixes #3604

### CODE REVIEW

**Review by juliushaertl — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by GretaD** [comment]
will rebase after #3586  is merged

**Comment by ChristophWurst** [comment]
> will rebase after #3586 is merged

go :)

**Comment by ChristophWurst** [comment]
It works but I can't scroll again :/

**Comment by GretaD** [comment]
> It works but I can't scroll again :/

fixed

File: src/components/ThreadEnvelope.vue [hunk 1]
```diff
@@ -346,5 +348,8 @@ export default {
 	.left {
 		flex-grow: 1;
 	}
+	::v-deep .modal-container {
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:351]: can we remove this duplicate code from `Thread.vue` then?

**Pull request merged**